examples/functions/functions.bal

```diff
@@ -1,15 +1,19 @@
 import ballerina/io;
 
-// This function takes a `string` argument, but does not have a return value.
+// This function takes in a `string` argument, but does not have a return
+// value.
 function printValue(string value) {
     io:println(value);
 }
 
-// This function takes two `int` values and returns their sum as an integer.
+// This function takes in two `integer` values as arguments and returns their
+// sum as an integer.
 function add(int a, int b) returns (int) {
     return a + b;
 }
 
+// This function named `main` is considered as the default function as an entry
+// point.
```

Comment:
@MaryamZi what would be the best way to phrase this?

Reply:
Shall we say
```ballerina
// A public function named `main` is considered a default entry point to a Ballerina program.
```